app/Http/Controllers/Apis/Protected/Summit/OAuth2SummitAttendeeNotesApiController.php

```diff
@@ -0,0 +1,282 @@
+     * @param $summit_id
+     * @param $attendee_id
+     * @param $note_id
+     * @return mixed
+     */
+    public function getAttendeeNote($summit_id, $attendee_id, $note_id) {
+        return $this->processRequest(function() use($summit_id, $attendee_id, $note_id) {
+            $summit = SummitFinderStrategyFactory::build($this->summit_repository, $this->resource_server_context)->find($summit_id);
+            if (is_null($summit)) return $this->error404();
+
+            $attendee = $summit->getAttendeeById(intval($attendee_id));
+            if (is_null($attendee)) return $this->error404("Attendee id {$attendee_id} not found in summit {$summit_id}.");
+
+            $attendee_note = $attendee->getNoteById(intval($note_id));
+            if (is_null($attendee_note)) return $this->error404("Attendee note id {$note_id} not found for attendee {$attendee_id}.");
+
+            return $this->ok
+            (
+                SerializerRegistry::getInstance()->getSerializer
+                (
+                    $attendee_note,
+                    SerializerRegistry::SerializerType_Private
+                )->serialize
+                (
+                    SerializerUtils::getExpand(),
+                    SerializerUtils::getFields(),
+                    SerializerUtils::getRelations(),
+                    ['serializer_type' => SerializerRegistry::SerializerType_Private]
+                )
+            );
+        });
+    }
+
+    /**
+     * @param $summit_id
+     * @param $attendee_id
+     * @return mixed
+     */
+    public function addAttendeeNote($summit_id, $attendee_id) {
+        return $this->processRequest(function() use($summit_id, $attendee_id) {
+            $summit = SummitFinderStrategyFactory::build($this->summit_repository, $this->resource_server_context)->find($summit_id);
+            if (is_null($summit)) return $this->error404();
+
+            $payload = $this->getJsonPayload(SummitAttendeeNoteValidationRulesFactory::buildForAdd(Request::all()));
+
+            $note = $this->attendee_service->upsertAttendeeNote($summit, intval($attendee_id), null, $payload);
+
+            return $this->created(SerializerRegistry::getInstance()->getSerializer($note)->serialize());
+        });
+    }
+
+    /**
+     * @param $summit_id
+     * @param $attendee_id
+     * @param $note_id
+     * @return mixed
+     */
+    public function updateAttendeeNote($summit_id, $attendee_id, $note_id) {
+        return $this->processRequest(function() use($summit_id, $attendee_id, $note_id) {
+            $summit = SummitFinderStrategyFactory::build($this->summit_repository, $this->resource_server_context)->find($summit_id);
+            if (is_null($summit)) return $this->error404();
+
+            $payload = $this->getJsonPayload(SummitAttendeeNoteValidationRulesFactory::buildForUpdate(Request::all()));
+
+            $note = $this->attendee_service->upsertAttendeeNote($summit, intval($attendee_id), intval($note_id), $payload);
```

Comment:
u need to get here the current user and pass it as the author

Reply:
done